joplin/templates/wagtailadmin/shared/sidebar-preview.html

```diff
@@ -1,19 +1,17 @@
-<div class="sidebar-container">
+<div id="mobile-preview-sidebar" class="sidebar-container">
     <div class="fixed-content">
         <div class="thumbnail-container">
             <div class="thumbnail">
-                <iframe name="sidebar-iframe" frameborder="0" onload="this.style.opacity = 1" src="{{style_guide_url}}"></iframe>
+                <iframe id="mobile-preview-iframe" name="sidebar-iframe" frameborder="0" onload="this.style.opacity = 1" src="{{style_guide_url}}"></iframe>
             </div>
         </div>
         <div class="coa-sidebar-buttons">
-            <a class="coa-sidebar-button js-toggle-active-panel"
-                {% if page.janis_preview_url %} 
-                    href="{{ page.janis_preview_url }}" 
-                {% else %}
-                    {# TODO: When we are creating a page (as opposed to editing), we do not yet have a saved revision so what do we show in the mobile preview? #}
-                    href="https://alpha.austin.gov" 
-                {% endif %}
-                target="sidebar-iframe"
+            <a id="mobile-preview-sidebar-button" class="coa-sidebar-button js-toggle-active-panel"
+              {# Placeholder that will be overwritten by changeLanguage() in editor.js #}
+              {# When editing an existing page, there will be a mobile preview #}
+              {# However, there will not be a mobile preview yet for a new page #}
```

Comment:
I don't think we actually have "new" pages in the sense wagtail thinks of them. Our create content modal adds some basic data to a page and then saves it before redirecting to a "edit" page screen

Reply:
Huh, okay. I'll take out my misleading comments.